exec/java-exec/src/test/java/org/apache/drill/TestUnionAll.java

```diff
@@ -527,6 +532,52 @@ public void testUnionAllRightEmptyJson() throws Exception {
       .build().run();
   }
 
+  @Test
+  public void testUnionAllLeftEmptyJson() throws Exception {
```

Comment:
Are there any union-all tests with  WHERE 1 = 0 filters on the left and/or right inputs ?  If not, could you add such tests ?  We should distinguish between a truly empty input vs. one that occurs as a result of a False filter.